CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
+## v1.0.0-rc.6
```

Comment:
should we keep rc here? or release v1 as well :eyes:

Reply:
Ideally the switch between last rc and final release does not have any changes to signal that rc becomes release. So I would release rc.6 and then release v1 afterwards.